pkg/operator/controller/garden/garden/components.go

```diff
@@ -1298,6 +1298,12 @@ func (r *Reconciler) newBlackboxExporter(garden *operatorv1alpha1.Garden, secret
 		}
 	}
 
+	isDashboardCertificateIssuedByGardener := false
+	if wildcardCertSecretName == nil {
+		// See function 'gardenerDashboard.ingress(ctx)' in pkg/component/gardener/dashboard/ingress.go
+		isDashboardCertificateIssuedByGardener = true
+	}
```

Comment:
```suggestion
	// See function 'gardenerDashboard.ingress(ctx)' in pkg/component/gardener/dashboard/ingress.go
	isDashboardCertificateIssuedByGardener := wildcardCertSecretName == nil
```

Reply:
Thank you! I amended the last commit with the suggestion above.